website/docs/addon_applications_admin.md

```diff
@@ -60,4 +70,37 @@ In the image before you can see that we set most of the environment variables in
 In this example MTOA will automatically set the `MAYA_VERSION`(which is set by Maya Application environment) and `MTOA_VERSION` into the `MTOA` variable. We then use the `MTOA` to set all the other variables needed for it to function within Maya.
 ![tools](assets/settings/tools_01.png)
```

Comment:
Unfortunately I can't comment on the exact line - but can we use this PR to also refactor `pype` on [Line 67](https://github.com/ynput/ayon-documentation/blob/2f272ba7d4ff8d73262a7e7d095c2b962a98e3b8/website/docs/addon_applications_admin.md?plain=1#L67) to `AYON`?